src/Storage/RegressionTests/adls_setaclresusive.ps1

```diff
@@ -50,7 +50,7 @@ BeforeAll {
     $ctx2 = New-AzStorageContext  $testNode.accountName
 
     $filesystemName = "adlstest2"
-    $localSrcFile = ".\data\testfile_1024K_0" #The file needs to exist before tests, and should be 512 bytes aligned
```

Comment:
The line #46-47 also can't pass on my machine. 
Following can pass

$storageAccountName = $testNode.accountName
$storageAccountKey = (Get-AzStorageAccountKey -ResourceGroupName $resourceGroupName -Name $storageAccountName)[0].Value

Reply:
It will work with the updated config file

Reply:
Hmm actually let me make the change as this info is actually redundant in the config